diagnostic.md

```diff
@@ -94,4 +94,4 @@ var teamMember = {
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of
 `teamMember['name']['surname']`?
 
-_Replace this text with your answer._
+exercise of Jason Weeks of Jason Weeks Weeks
```

Comment:
The value of teamMember['hobby'] is "exercise".
The value of teamMember['name'] is "{ given: 'Jason', surname: 'Weeks' }".
The value of teammember['name']['surname'] is "Weeks".